include/quadrilateral_element.tcc

```diff
@@ -323,14 +323,12 @@ inline std::vector<Eigen::MatrixXd>
 
   // Gradient shapefn of the cell
   // dN/dx = [J]^-1 * dN/dxi
-  Eigen::MatrixXd grad_shapefn = grad_sf * jacobian.inverse();
+  Eigen::MatrixXd grad_shapefn = grad_sf * (jacobian.inverse()).transpose();
```

Comment:
Can you change this to`Eigen::MatrixXd grad_shapefn = jacobian.inverse() * grad_sf.transpose();`